README.md

```diff
@@ -154,8 +154,8 @@ Other Style Guides
     const item = {};
     ```
 
-  <a name="es6-computed-properties"></a><a name="3.4"></a>
-  - [3.4](#es6-computed-properties) Use computed property names when creating objects with dynamic property names.
+  <a name="es6-computed-properties"></a><a name="3.2"></a>
```

Comment:
Thanks for pointing this out @ljharb. Does this mean that you would prefer to keep the numbering as is (out of sequence), or update the numbering but leave the links alone?